### ISSUE TITLE
"ERR command is not allowed" when using RedisCluster against clustered CRDB with OSS API

### ISSUE THREAD

**Issue by mruki** [sewenew/redis-plus-plus#71]
Hi,

thanks for the library - it is really useful and neatly written.
I'm testing it currently against CRDB with OSS cluster API on Redis Enterprise (version 5).
Problem is, with RedisCluster class used for connections I'm getting "ERR command is not allowed" error.
After quick googling I got to know that it's due to not supporting the OSS API by the client.
It seems to work fine, when using Redis class for connections, but then I will loose all of the cluster features, like ASK / MOVE redirections, so that doesn't make sense.

Any ideas here?

Thanks,
Konrad

**Comment by sewenew** [comment]
Hi @mruki 

>  I got to know that it's due to not supporting the OSS API by the client.

Sorry, but I didn't hear about the OSS API before. I did some research, and it seems this API only works with Redis Enterprise version. Also, I didn't find any documentation about the details of OSS API. So far, redis-plus-plus cannot support OSS API. If you have any documentation about this API, you can share the link to me, and I'll take a look.

Regrads

**Comment by mruki** [comment]
Thanks @sewenew 
It was a surprise for me too - I was able to connect to the same cluster with other library (Redis ACL) without this error.
ACL library has some drawbacks though, that's why I started exploring your library, which seems much better.
I will try to reach out to Redis Labs to get a better description of this API, as this is really a novelty to me too.
Frankly I don't really understand why I get this error here, it works fine with a python client and Java (lettuce).
Redis Labs makes things really complicated...

BR,
Konrad
